docs/static/filebeat_modules/mysql/error/pipeline.conf

```diff
@@ -0,0 +1,34 @@
+input {
+  beats {
+    # The port to listen on for filebeat connections.
+    port => 5044
+    # The IP address to listen for filebeat connections.
+    host => "0.0.0.0"
+  }
+}
+filter {
+   grok {
+      match => { "message" => [""%{LOCALDATETIME:[mysql][error][timestamp]} (\[%{DATA:[mysql][error][level]}\] )?%{GREEDYDATA:[mysql][error][message]}",
```

Comment:
This config is invalid. Need to remove extra quotation so that you have `["%{LOCALDATETIME:[mysql][error][timestamp]`